### DISCUSSION TITLE
Using with other SEO plugin

### DISCUSSION THREAD

**Discussion by liamodowd** [wolfdevsllc/og-image-overlay#3] [Ideas]
Great bit of code, can you clarify how we use with another SEO plugin? (Using SEOPress)

**Comment by itsmereal** [comment]
That can be done in various ways. Each plugin is different and they might have different hooks or filters to modify and make it work. It really depends on what plugin you are using. For starters, you may check how the system actually works and then proceed with customization. 

Here is the full tutorial of how this works: https://itsmereal.com/automated-open-graph-image-overlay-for-wordpress/
